### PR TITLE
Class branch

### PR DESCRIPTION
**Pull request by deikdesign**
- Using classes on the project
- Using objects and arrays inside a class
- Add and remove books feature

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable prefer-const */
+/* eslint-disable radix */
+/* eslint-disable max-classes-per-file */
```
Comment (bimbolabuari) [inline script.js]: Kindly remove these disabled linters, linting improves readability, removes errors. 👍

Please check for other occurrences 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hello @deikdesign , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Used JavaScript classes 👍 
- Created class methods to add and remove books. 🚀 
- Maintained all of the original functionality of the application ✊ 
- Used only plain JavaScript. 🎉 
 
There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: script.js (hunk 1 above)
Comment (danalvrz) [reply script.js]: I made the changes, I only disabled the two that are mention in the project instructions:

TIP: If you have issues with the linter for either having multiple classes at the same file, or not an unused-variable error if you separated the code in multiple files, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */ .

**Comment by danalvrz** [comment]
I made the changes, I only disabled the two that are mention in the project instructions:

TIP: If you have issues with the linter for either having multiple classes at the same file, or not an unused-variable error if you separated the code in multiple files, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */ .

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @deikdesign, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**